compilers/concrete-compiler/compiler/lib/Support/CompilationFeedback.cpp

```diff
@@ -93,6 +93,15 @@ llvm::json::Value toJSON(const mlir::concretelang::CompilationFeedback &v) {
       {"crtDecompositionsOfOutputs", v.crtDecompositionsOfOutputs},
   };
 
+  auto memoryUsageJsonArray = llvm::json::Array();
```

Comment:
why using an array here, instead of an object with key=location, and value an object that contains the memory footprint?

```
{
  ...,
  "memroyUsagePerLoc": {
    "loc1": {"footprint": 12},
    ...
  }
}
```